services/horizon/internal/actions_effects.go

```diff
@@ -77,6 +77,20 @@ func (action *EffectIndexAction) SSE(stream sse.Stream) {
 	)
 }
 
+// GetTopic is a method for actions.SSE
+func (action *EffectIndexAction) GetTopic() string {
+	if action.GetString("account_id") != "" {
+		return action.GetString("account_id")
+	} else if action.GetString("ledger_id") != "" {
+		return action.GetString("ledger_id")
+	} else if action.GetString("tx_id") != "" {
+		return action.GetString("tx_id")
+	} else if action.GetString("op_id") != "" {
+		return action.GetString("op_id")
```

Comment:
- Can there be a case where the action has more than one field return a non-empty string? If so there's a potential bug here because for example it gives `account_id` priority over `op_id`.
- this pattern is repeated in other functions in this pr